include/os/linux/spl/sys/kmem.h

```diff
@@ -127,21 +127,12 @@ spl_fstrans_check(void)
 	return (current->flags & SPL_FSTRANS);
 }
 
-#ifdef HAVE_ATOMIC64_T
 #define	kmem_alloc_used_add(size)	atomic64_add(size, &kmem_alloc_used)
 #define	kmem_alloc_used_sub(size)	atomic64_sub(size, &kmem_alloc_used)
 #define	kmem_alloc_used_read()		atomic64_read(&kmem_alloc_used)
 #define	kmem_alloc_used_set(size)	atomic64_set(&kmem_alloc_used, size)
 extern atomic64_t kmem_alloc_used;
-extern unsigned long long kmem_alloc_max;
-#else  /* HAVE_ATOMIC64_T */
-#define	kmem_alloc_used_add(size)	atomic_add(size, &kmem_alloc_used)
-#define	kmem_alloc_used_sub(size)	atomic_sub(size, &kmem_alloc_used)
-#define	kmem_alloc_used_read()		atomic_read(&kmem_alloc_used)
-#define	kmem_alloc_used_set(size)	atomic_set(&kmem_alloc_used, size)
-extern atomic_t kmem_alloc_used;
-extern unsigned long long kmem_alloc_max;
-#endif /* HAVE_ATOMIC64_T */
```

Comment:
Let's go a bit father, we can drop the wrappers entirely and directly call the `atomic64_*` version of the function.  The wrappers were originally needed because very, very old versions of the kernel (2.6 era) didn't provide a 64-bit atomic type.

Reply:
Thanks, I wanted to but didn't know if the names were important to someone. Done!